woocommerce/woocommerce-gutenberg-products-block
# Updated composer.lock and minor deps

## What
Updated composer.lock file. Some minor updates were also made:

- Upgrading automattic/jetpack-admin-ui (v0.2.23 => v0.2.25)
- Upgrading automattic/jetpack-connection (v1.58.2 => v1.60.1)
- Upgrading automattic/jetpack-status (v1.18.5 => v1.19.0)
- Upgrading theseer/tokenizer (1.2.1 => 1.2.2)

## Why

composer.lock was out of date with composer.json



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure tests pass, smoke test the extension monitoring error logs (js & PHP)


## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> n/a
